### ISSUE TITLE
Question regarding call for entire board to resign

### ISSUE THREAD

**Issue by phikal** [rms-open-letter/rms-open-letter.github.io#1033]
Hi,

I'd like to begin by making clear that while I disagree with the position of this letter (his [explicitly stated political positions](https://stallman.org/articles/necessary-changes.html) appear to directly contradict the raised accusations), I anticipate that you will "win" in the end, as is usual in our times.

Nevertheless, the passage 

> We are calling for the removal of the entire Board of the Free Software Foundation.

is distressing. Could this be elaborated? What would this imply? Are you in effect calling for the dissolution of the Free Software Foundation as such?

If this were to happen, it seems there would be difficult to maintain the legitimacy of the FSF. I fear the repercussions of these brash actions could lead to more harm being done for user freedom, than if a compromise were to be attempted. I don't hope that this is the intended consequence of the letter.

I will naively end this post with a plea for the editors and the people who signed the letter to reconsider the possible repercussions, regardless of their subjective experiences on the topic. Neither the status quo nor the actions proposed here seem to offer real, long-term solutions. There is no simple solution to this issue, but I am certain that it can be found!

Yours truly and sincerely,
Philip K.

**Comment by fa7ad** [comment]
To me, this means replacing the members on the board of FSF rather than dissolving the organization.

My guess is that this is the intent of the authors as well.

Remember, all this is happening because the board was complacent enough with RMS's behaviour to bring him back on board.

**Comment by mollydb** [comment]
The FSF would not be the first Board to have a complete turnover. The Tor Project board of directors resigned and a new board was elected several years ago: https://blog.torproject.org/tor-project-elects-new-board-directors. I'm sure others have as well, even outside of the tech nonprofit space.

**Comment by phikal** [comment]
> Remember, all this is happening because the board was complacent enough with RMS's behaviour to bring him back on board.

To my knowledge, there has been no official statement on the issue yet. All I know of is the short video from LibrePlanet and that the [Board section](https://www.fsf.org/about/staff-and-board) on the FSF homepage has been updated. 

> The Tor Project board of directors resigned and a new board was elected several years ago: https://blog.torproject.org/tor-project-elects-new-board-directors.

Interesting, I did not know about that. Does anyone know how that turned out?

What I, and I supposed many others, fear is how the ideological integrity can be preserved. The FSF currently holds a very strong  but controversial position on topics from licensing over privacy to user freedom and respect. There are many people that are interested in getting rid of such a player in the tech world or replacing it with a more timid and "compromising" alternative. 

For me, RMS and the FSF stands for the opposition to this kind of corruption. Without it, the FSF would loose all meaning.
